src/components/SearchResults.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "../styles/searchResults.css";
+import PropTypes from "prop-types";
+
+// eslint-disable-next-line react/function-component-definition
```

Comment:
This eslint escape is used in a few places. Rather than doing this would it not be better to a a rule to your `eslintrc.json` to catch all instances of this and make your code more uniform across the board? You could do this by adding this to the `rules` object in your eslintrc:
```
"react/function-component-definition": [
      1,
      {
        "namedComponents": "arrow-function"
      }
    ]
```
This will require you to use arrow functions for your component definitions and negate the need for this comment. (You will also have to update your `App.js` as that currently uses a function declaration)